ocaml/testsuite/tests/jane-modular-syntax/user_error7.ml

```diff
@@ -2,21 +2,32 @@
    * expect
 *)
 
-(* If you don't use these as applications, they don't pass through the modular
-   extensions machinery and fail with a normal OCaml error *)
+(* If we use extension nodes outside of the context they are expected,
+   they aren't interpreted by the modular syntax machinery and fail with
+   a normal OCaml error *)
```

Comment:
Should we also have this test for attributes?

Reply:
We most definitely should. I'm adding this to a stacked PR where I add `erasable` and `non_erasable`, as it's slightly more convenient there.